scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementSample.java

```diff
@@ -0,0 +1,205 @@
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties.SurfacePlacement;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+import com.esri.arcgisruntime.symbology.TextSymbol.HorizontalAlignment;
+import com.esri.arcgisruntime.symbology.TextSymbol.VerticalAlignment;
+
+public class SurfacePlacementSample extends Application {
+
+  private SceneView sceneView;
+  private GraphicsOverlay drapedBillboardedOverlay;
+  private GraphicsOverlay drapedFlatOverlay;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Surface Placement Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createImagery());
+
+      // add the SceneView to the stack pane
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // add a camera and initial camera position
+      Camera camera = new Camera(53.05, -4.01, 1115, 299, 88, 0);
+      sceneView.setViewpointCamera(camera);
+
+      // add base surface for elevation data
+      Surface surface = new Surface();
+      surface.getElevationSources()
+          .add(new ArcGISTiledElevationSource(
+              "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+      scene.setBaseSurface(surface);
+
+      // create overlays with elevation modes
+      drapedBillboardedOverlay = new GraphicsOverlay();
+      drapedBillboardedOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.DRAPED_BILLBOARDED);
+
+      drapedFlatOverlay = new GraphicsOverlay();
+      drapedFlatOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.DRAPED_FLAT);
+
+      GraphicsOverlay relativeOverlay = new GraphicsOverlay();
+      relativeOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.RELATIVE);
+
+      GraphicsOverlay absoluteOverlay = new GraphicsOverlay();
+      absoluteOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.ABSOLUTE);
+
+      // create point for graphic location
+      Point point = new Point(-4.04, 53.06, 1000, camera.getLocation().getSpatialReference());
```

Comment:
since you don't explicitly set the spatial reference of the camera, better to just set it to WGS84.